internal/manager/background.go

```diff
@@ -64,38 +65,46 @@ func checkChanges(m *Manager) {
 func processLocalChanges(m *Manager) {
 	for {
 		time.Sleep(processInterval)
-		processChanges(m)
+		processChanges(m, false)
 	}
 }
 
-func processChanges(m *Manager) {
-	items := m.tracker.Flush()
+func processChanges(m *Manager, forceAll bool) {
```

Comment:
@paddlesteamer I think it's best to avoid boolean function arguments, as it becomes very difficult to understand what the flag means while reading the parts that the function is being used.

If you take a look at `manager.go` where this is called from, you'll see what I mean.

Reply:
I think you're right. I just wanted to use tiny little boolean parameter which came very handy :D What do you suggest instead of this? Two separate functions maybe?

Reply:
Aa maybe using flags will do. Like processChanges(m, forceAll)

Reply:
@paddlesteamer Two functions make a lot of sense. An easier alternative is to define 2 constants (the good thing is you can add more later) and pass those instead. For instance;

```go
processChanges(manager, ForceAll);
processChanges(manager, OnlySelected); // just made this up, might not make sense
...
```

Reply:
:+1: